.travis.yml

```diff
@@ -16,7 +17,15 @@ jobs:
   - stage: Integration tests
     script:
     - sudo /etc/init.d/mysql stop
+    - sudo sysctl fs.inotify.max_user_watches=524288; sudo sysctl -p
     - git clone https://github.com/streamr-dev/streamr-docker-dev.git
+    - sudo ifconfig docker0 10.200.10.1/24
+    - "$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh start 4"
+    - "$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh start engine-and-editor"
+    - while true; do http_code=$(curl -s -o /dev/null -I -w "%{http_code}" http://localhost:8081/streamr-core/login/auth); if [ $http_code = 200 ]; then echo "EE up and running"; break; else echo "EE not receiving connections"; sleep 5s; fi; done
     - "$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh start 5"
```

Comment:
This line should also be unnecessary, `start engine-and-editor` has already started the same set of services as `start 5`?

Actually, to be semantically correct and make the simplest statement that `streamr-client` requires all services to be running, we could replace all the other start commands with just `"$TRAVIS_BUILD_DIR/streamr-docker-dev/streamr-docker-dev/bin.sh start 5"`, and then have the wait loops that check that engine-and-editor and data-api are up?